packages/runtime-core/src/components/KeepAlive.ts

```diff
@@ -300,10 +300,11 @@ export const KeepAlive = (KeepAliveImpl as any) as {
 }
 
 function matches(pattern: MatchPattern, name: string): boolean {
+  name = name.trim()
   if (isArray(pattern)) {
     return pattern.some((p: string | RegExp) => matches(p, name))
   } else if (isString(pattern)) {
-    return pattern.split(',').indexOf(name) > -1
+    return pattern.split(',').map((i: string) => i.trim()).indexOf(name) > -1
```

Comment:
if there is a `map` we should directly use `some()` instead of `indexOf() > -1`.

Reply:
Sure, make sense, will do that.